# Lab6_Part_2-Hypothesis Testion

Choose the best Test and apply.

### Question1:

suppose we want to know whether or not the mean weight of a certain species of some turtle is equal to 310 pounds.

To test this, we go out and collect a simple random sample of turtles with the following weights:

Weights: 300, 315, 320, 311, 314, 309, 300, 308, 305, 303, 305, 301, 303

In [1]:
import numpy as np
from scipy import stats

weights = [300, 315, 320, 311, 314, 309, 300, 308, 305, 303, 305, 301, 303]

mu = 310
t_statistic, p_value = stats.ttest_1samp(weights, mu)
t_statistic, p_value

(-1.5848116313861254, 0.1389944275158753)

- the t-statistic of -1.585 indicates that the sample mean is 1.585 standard errors below the hypothesized mean of 310 pounds.
- The p-value of 0.139 means there is a 13.9% chance of observing a sample mean as extreme as 300.923 (the sample mean) if the true population mean were 310 pounds.

### Conclusion
There is not enough evidence to conclude that the mean weight of the turtle species is different from 310 pounds.

### Question2:

suppose we want to know whether or not the mean weight between two different species of turtles is equal.

To test this, we collect a simple random sample of turtles from each species with the following weights:

Sample 1: 300, 315, 320, 311, 314, 309, 300, 308, 305, 303, 305, 301, 303

Sample 2: 335, 329, 322, 321, 324, 319, 304, 308, 305, 311, 307, 300, 305

In [2]:
sample1 = [300, 315, 320, 311, 314, 309, 300, 308, 305, 303, 305, 301, 303]
sample2 = [335, 329, 322, 321, 324, 319, 304, 308, 305, 311, 307, 300, 305]

t_statistic, p_value = stats.ttest_ind(sample1, sample2)
t_statistic, p_value

(-2.10090292575557, 0.04633501389516513)

### Conclusion
There is enough evidence to conclude that the mean weights of the two turtle species are different

### Question3:


suppose we want to know whether or not a certain training program is able to increase the max vertical jump (in inches) of basketball players.

The following data shows the max jump height (in inches) before and after using the training program for each player:

Before: 22, 24, 20, 19, 19, 20, 22, 25, 24, 23, 22, 21

After: 23, 25, 20, 24, 18, 22, 23, 28, 24, 25, 24, 20

In [3]:
before = [22, 24, 20, 19, 19, 20, 22, 25, 24, 23, 22, 21]
after = [23, 25, 20, 24, 18, 22, 23, 28, 24, 25, 24, 20]
t_statistic, p_value = stats.ttest_rel(before, after)
t_statistic, p_value

(-2.5289026942943655, 0.02802807458682508)

### Conclusion
There is enough evidence to conclude that the training program has significantly increased the max vertical jump of the basketball players. ​